src/openrct2/ride/Ride.cpp

```diff
@@ -7081,23 +7080,16 @@ void sub_6CB945(Ride* ride)
                     if (!ride_get_exit_location(ride, stationId).isNull())
                         break;
 
-                    ride_set_exit_location(
-                        ride, stationId,
-                        { location.x / 32, location.y / 32, ride->stations[stationId].Height,
-                          (uint8_t)tileElement->GetDirection() });
+                    CoordsXYZD loc = { location, ride->stations[stationId].GetBaseZ(), (uint8_t)tileElement->GetDirection() };
```

Comment:
While you're at it, do `static_cast<Direction>(tileElement->GetDirection())` instead. I've been trying to get away with these C casts.

Reply:
GetDirection() already returns a `Direction`, so casting that to `Direction` is a bit pointless.

Reply:
Even better, we can drop the casts and be happy :)